Source/Initialization/WarpXInitData.cpp

```diff
@@ -416,9 +416,9 @@ WarpX::InitLevelData (int lev, Real /*time*/)
     }
 
     if (costs[lev]) {
-        std::fill((*costs[lev]).begin(),
-                  (*costs[lev]).end(),
-                  0.0);
+        for (int i : costs[lev]->IndexArray()) {
+            (*costs[lev])[i] = 0.0;
+        }
```

Comment:
Does this object support
```suggestion
        for (auto& c : costs[lev]) {
            c = 0.0;
        }
```
?

Reply:
I think it's not supported, unfortunately.  Maybe worth an AMReX PR?